### DISCUSSION TITLE
[Hackathon Use Case] - Create a Reference Architecture for Open RegTech Reporting

### DISCUSSION THREAD

**Discussion by mcleo-d** [finos/finos-pmcs#252] [Ideas]
# Create a Reference Architecture for Open RegTech Reporting

DR covers several trade & transaction reporting regimes that have been developed through various implementation initiatives, each at different stages of maturity.

During the FINOS hackathon the Open RegTech SIG wants to created a reference architecture for Open RegTech Reporting that ...

## Benefits
- Share development of rules that lead to improved understanding of regulations and a reduced operational risks
- Data reuse across regulations
- Reuse of data controls with common provisioning
- Technology architecture/capabilities developed leads to cost savings

## An Example FINOS Reference Architecture
The following illustrates what the Open RegTech Reporting reference architecture could look like.

![Screenshot 2023-04-06 at 14 43 28](https://user-images.githubusercontent.com/6029572/230396452-e41b4284-8182-4bb4-9c7e-45581330a3e9.png)

## Get Involved

Please join this use case to drive the delivery of an Open RegTech reference architecture, to learn more about FINOS Open RegTech projects and to contribute to a practical FINOS Open RegTech reference architecture.

## Useful apps, systems, projects, technologies, libraries and datasets

- [Open RegTech](https://www.finos.org/open-regtech)
- [Legend](https://legend.finos.org)
- [Morphir](https://morphir.finos.org)
- [CDM](https://github.com/finos/common-domain-model)

**Comment by TheJuanAndOnly99** [comment]
Closing discussion in preparation [for this years hackathon](https://www.finos.org/hosted-events/2024-10-25-citi-india-hackathon-supported-by-finos). Feel free to reopen and update if its still relevant.
